lib/housekeeping.js

```diff
@@ -38,17 +38,109 @@ class HouseKeeper {
     this.tagger = tagger;
   }
 
-  async _handleVolumeData(volume) {}
+  _calculateVolumeTotals(volumes) {}
+
+  _submitVolumeTotals(totalsByCategory) {
+    // Assumption:
+    // totalsByCategory is organized as follows:
+    // { 
+    //   ${region}: [ 
+    //     {
+    //       ${vol-type}: {
+    //         active: {
+    //           gb: value,
+    //           count: value
+    //         },
+    //         unused: {
+    //           gb: value,
+    //           count: value
+    //         }
+    //       }
+    //     },
+    //     ...
+    //   ]
+    // }
+
+    //assert(typeof totalsByCategory === 'object');
+
+    //assert(Object.keys(totalsByCategory).length === 1);
+    let region =  Object.keys(totalsByCategory)[0];
+    
+    let volumeTypes = totalsByCategory[region];
+    //assert(typeof volumeTypes === 'object');
+    //assert(typeof volumeTypes.hasOwnProperty('length'));
+
+    volumeTypes.forEach(function(data) {
+      //assert(typeof data === 'object');
+      
+      //assert(Object.keys(data).length === 1);
+      let type = Object.keys(data)[0];
+
+      //assert(data.hasOwnProperty('active'));
+      //assert(data.hasOwnProperty('unused'));
+      let active = data.active;
```

Comment:
This can be made a bit nicer with 

`let {active, unused} = data;`

if you'd like!

Reply:
I'd forgotten about that syntax, and it's definitely cleaner! Thanks!